client/components/checkbox/style.scss

```diff
@@ -0,0 +1,52 @@
+.checkbox {
+	cursor: pointer;
+	display: block;
+	font-size: 14px;
+	margin-bottom: 5px;
+	position: relative;
+	user-select: none;
+}
+
+.checkbox input[type="checkbox"] {
+	@include hide-content-accessibly;
+	border: 0;
+	margin: -1px;
+	padding: 0;
+}
+
+.checkbox .gridicon {
+	fill: $blue-medium;
+	left: 10px; // half width of box plus 1px of border
+	position: absolute;
+	top: 50%;
+	transform: translate(-50%, -50%);
+	z-index: 1;
```

Comment:
@drw158 or @folletto could one of you review this `z-index`? I can't remember for sure but I thought that we were making some kind of standardization for declaring these and this is a low-level application component.